packages/create-sitecore-jss/src/templates/angular/src/app/jss-graphql.module.ts

```diff
@@ -3,10 +3,8 @@ import { HttpClientModule, HttpHeaders } from '@angular/common/http';
 import { InMemoryCache, NormalizedCacheObject, PossibleTypesMap } from '@apollo/client/core';
 import { Apollo, ApolloModule } from 'apollo-angular';
 import { HttpBatchLink } from 'apollo-angular/http';
-import { createPersistedQueryLink } from 'apollo-angular/persisted-queries';
```

Comment:
Remove `apollo-angular/persisted-queries` and `sha256` dependencies from package.json?

Reply:
Removed `sha56` but `persisted-queries` is a submodule of `apollo-angular` which is the primary dependency for apollo.